apps/L1G/tinder/tinder-frontend/src/app/(main)/home/page.tsx

```
@@ -10,6 +10,7 @@ const HomePage = () => {
   const [like] = useLikeMutation();
   const [dislike] = useDislikeMutation();
   const [currentIndex, setCurrentIndex] = useState(0);
+  const [matchedProfile, setMatchedProfile] = useState<UserProfile | null>(null);
```

ESLint

[error] 'matchedProfile' is assigned a value but never used.

(no-unused-vars)